### PR TITLE
Doc updates for Authtoken and IPv6

### PR DESCRIPTION
**Pull request by aaha97**
Add instructions for authtoken encoding
Fix wording for IPv6 docs

### CODE REVIEW

File: docs/values.md [hunk 1]
```diff
@@ -226,6 +226,20 @@ The generated authtoken from the Avi controller can be specified with this flag
 and maintained by secret object. The token refresh is managed by AKO. In case of token refresh failure, a new token needs to be generated and updated
 into the secret object.
 
+To encode a token generated from controller to directly patch `avi-secret`, use
```
Comment (akshayhavile) [inline docs/values.md:229]: Our intention here is to give few ways to encode token properly.  Should we rephrase it as: `Few ways, to encode controller generated token properly, are 
1. shell
2. Python` ?

Comment (aaha97) [reply docs/values.md]: yes, rephrased the sentence

**Review by akshayhavile — DISMISSED** [review]

**Review by swathinsankaran — DISMISSED** [review]
LGTM

**Comment by aaha97** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Review by swathinsankaran — APPROVED** [review]

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
@aaha97 could you please rebase this PR?

**Comment by swathinsankaran** [comment]
build ako

**Review by swathinsankaran — APPROVED** [review]

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Comment by akshayhavile** [comment]
build ako

**Pull request merged**